packages/repmgr13/README

```diff
@@ -0,0 +1,7 @@
+Build with:
+
+Fetch source files from https://rpm-manageiq-org.nyc3.digitaloceanspaces.com/index.html?prefix=sources_cache/repmgr13/ if needed.
+
+```bash
+mock --spec=./repmgr.spec -r centos-stream+epel-8-x86_64 --sources=./
```

Comment:
How do I run this locally?

Reply:
You will need a CentOS VM with the rpm build tools installed.

Reply:
🤷 I don't see the point of that.   Will it work with any centos/rhel version with any version of mock or other dependencies?  The Dockerfile provides the recipe for a repeatable build environment and gets us away from local build machines.

Reply:
I know I'm missing the benefits (other than the sweet sweet deletions)... which is why I'm asking.  I can download the missing source files for repmgr and get rid of the hack to download the source during build if desired:

```
--undefine=_disable_source_fetch \
```

Reply:
Aside from the sweet sweet deletions 🤣...
- The build environment setup is much simpler. On RHEL/CentOS `dnf -y install epel-release` then on RHEL/CentOS/Fedora `dnf -y install mock`.
- Switching releases is also much easier since you just select a different target config `-r centos-stream+epel-9-x86_64` to switch to CentOS 9. 
- Mock found other dependency issues in the RPM spec that were not found with rpmbuild in the container.

Reply:
Also, I don't want to be in the business of maintaining a Dockerfile for every RPM that we build.

Reply:
🤷 I can merge but I'll probably end up resurrecting it locally when I need to do this again in several months.